src/main/steps/urls.ts

```diff
@@ -20,6 +20,11 @@ export const ACCESSIBILITY_STATEMENT: PageLink = '/accessibility-statement';
 export const TERMS_AND_CONDITIONS: PageLink = '/terms-and-conditions';
 export const CONTACT_US: PageLink = '/contact-us';
 
+/**   @C100 features */
+/* This is a route for the C100 application. */
+export const C100_URL: PageLink = '/c100-rebuild';
+export const C100_CONFIDENTIALITY_START_ALTERNATIVE: PageLink = `${C100_URL}/confidentiality/details-know`;
```

Comment:
use `C100_CONFIDENTIALITY_DETAILS_KNOW` this instead of 'C100_CONFIDENTIALITY_START_ALTERNATIVE'